django/contrib/gis/geos/prototypes/io.py

```diff
@@ -114,17 +114,10 @@ def __init__(self):
         self.ptr = self._constructor()
         # Loading the real destructor function at this point as doing it in
         # __del__ is too late (import error).
-        self._destructor.func = self._destructor.get_func(
-            *self._destructor.args, **self._destructor.kwargs
+        self.destructor.func = self.destructor.get_func(
```

Comment:
Not sure about the status here given the referenced commit had to be reverted, but this at least needs a rebase to merge cleanly.